lib/internal/errors.js

```diff
@@ -1265,7 +1265,7 @@ E('ERR_INVALID_TUPLE', '%s must be an iterable %s tuple', TypeError);
 E('ERR_INVALID_URI', 'URI malformed', URIError);
 E('ERR_INVALID_URL', function(input) {
   this.input = input;
-  return `Invalid URL: ${input}`;
```

Comment:
Maybe we should add a comment here explaining why the url is merely assigned to the error and not in the message in case anyone tries to add it again for debuggability